src/coreComponents/constitutive/fluid/multifluid/MultiFluidBase.hpp

```diff
@@ -46,14 +47,14 @@ class MultiFluidBase : public ConstitutiveBase
    *
    * @note This puts an upper bound on memory use, allowing to optimize code better
    */
-  static constexpr integer MAX_NUM_COMPONENTS = 16;
+  static constexpr integer MAX_NUM_COMPONENTS = MultiFluidConstants::MAX_NUM_COMPONENTS;
```

Comment:
not sure this shortcut needs to be kept. not sure if it is necessary to have the same litteral value stored in the binary 2 times.